### ISSUE TITLE
Please add XDEBUG for debugging

### ISSUE THREAD

**Issue by meglio** [vccw-team/vccw#79]
It seems as if xdebug is not installed and activated by default. Please could you add it?

**Comment by miya0001** [comment]
Thanks!
I released 2.12.0.
https://github.com/vccw-team/vccw/releases/tag/2.12.0

**Comment by chrispage1** [comment]
Hi,

For proper remote debugging, the following should be added to /etc/php.d/xdebug.ini or /etc/php.ini

```
xdebug.remote_enable=true
xdebug.remote_port=9000
xdebug.profiler_enable=1
xdebug.idekey=VCCWDEBUG
xdebug.remote_connect_back=true
```

Please could we get these added in? We could then listen remotely for connections with the key *VCCWDEBUG*. Would be great if you could release a fix with this :+1:
